ArchUnitNET/Domain/MethodMember.cs

```diff
@@ -38,6 +39,7 @@ public class MethodMember : IMember
         public IType ReturnType => _returnTypeInstance.Type;
         public bool IsStub { get; }
         public bool IsCompilerGenerated { get; }
+        public bool? IsIterator { get; }
```

Comment:
can we make this bool instead of bool? what does null mean in this context?

Reply:
it is null if the methodreference can't be resolved by mono